src/js/features/clear-clutter.js

```diff
@@ -13,4 +13,9 @@ module.exports = function() {
         removeElement('#nav_related_streams');
         $('body').append('<style>#nav_games, #nav_streams, #nav_related_streams, .js-recommended-channels { display: none !important; }</style>');
     }
+
+    // if setting "Hide Last Broadcast Banner" is true
+    if (bttv.settings.get('hideLastBroadcast') === true) {
```

Comment:
this is not the correct location for this.

additionally, we do not place css within js any longer.